buildstockbatch/test/test_inputs/enforce-validate-measures-bad-2.yml

```diff
@@ -1,17 +1,33 @@
-stock_type: residential
+schema_version: '0.3'
 buildstock_directory: test_openstudio_buildstock
 project_directory: project_singlefamilydetached
 weather_files_url: https://fake-url
 baseline:
-  n_datapoints: 30
   n_buildings_represented: 81221016
-  sampling_algorithm: quota
-residential_simulation_controls:
-  timesteps_per_hr: 4
-  begin_month: 1
-  begin_day_of_month: 1.5
-  end_month: 12
-  end_day_of_month: 31
+
+sampler:
+  type: residential_quota
+  args:
+    n_datapoints: 30
+
+workflow_generator:
+  type: residential_default
+  args:
+    residential_simulation_controls:
+      timesteps_per_hr: 4
+      begin_month: 1
+      begin_day_of_month: 1.5
+      end_month: 12
+      end_day_of_month: 31
+    timeseries_csv_export:
+      reporting_frequency: Huorly
```

Comment:
spelling? Or is that why this yml is bad? If so, then add comment saying that it is an intentional spelling mistake.

Reply:
This is intentionally misspelled. These files are for unit testing input validation and are not used by an end user.

Reply:
Can you add a comment saying `# intentionally misspelled for unit testing input validation` so that I don't make this same comment in a year :) ?